sensitivity-analysis-api/src/main/java/com/powsybl/sensitivity/SensitivityAnalysis.java

```diff
@@ -109,6 +135,32 @@ public SensitivityAnalysisResult run(Network network,
             return runAsync(network, factorsProvider, contingenciesProvider).join();
         }
 
+        public SensitivityAnalysisResult run(Network network,
+                                             String workingStateId,
+                                             SensitivityFactorsProvider factorsProvider,
+                                             SensitivityAnalysisParameters parameters,
+                                             ComputationManager computationManager) {
+            return runAsync(network, workingStateId, factorsProvider, new EmptyContingencyListProvider(), parameters, computationManager).join();
+        }
+
+        public SensitivityAnalysisResult run(Network network,
+                                             String workingStateId,
+                                             SensitivityFactorsProvider factorsProvider,
+                                             SensitivityAnalysisParameters parameters) {
+            return runAsync(network, workingStateId, factorsProvider, new EmptyContingencyListProvider(), parameters).join();
+        }
+
+        public SensitivityAnalysisResult run(Network network,
+                                             SensitivityFactorsProvider factorsProvider,
+                                             SensitivityAnalysisParameters parameters) {
+            return runAsync(network, factorsProvider, new EmptyContingencyListProvider(), parameters).join();
```

Comment:
These methods should call the corresponding `runAsync` method without adding new parameters